packages/main/src/plugin/container-registry.ts

```diff
@@ -290,12 +291,22 @@ export class ContainerProviderRegistry {
       internalProvider.api = undefined;
       internalProvider.libpodApi = undefined;
 
+      // if is the timeout running dont start other for events
+      // connect ENOENT \.\pipe\machine_name at PipeConnectWrap.afterConnect [as oncomplete]
+      if (this.activeTimeouts.get(containerProviderConnection.endpoint.socketPath)) {
```

Comment:
```suggestion
      if (this.activeTimeouts.has(containerProviderConnection.endpoint.socketPath)) {
```

would there be any difference of checking `get` vs `has` here ? The NodeJS.Timeout value will not change depending if the timeout has been executed or not.

![image](https://github.com/user-attachments/assets/8a80b4c7-a695-4298-a031-e3f9a8aed54e)